client/main.js

```diff
@@ -37,3 +38,13 @@ AutoForm.addHooks(['partyForm'],{
     FlowRouter.go('/party/' + this.docId);
   }
 });
+
+Meteor.subscribe('profiles');
```

Comment:
Thanks for reconfiguring the user ban! See my note in the Profiles publications file on this subscription.